src/app/api/activity/activityLogger.service.ts

```diff
@@ -0,0 +1,124 @@
+import { ActivityType, AssigneeType, Task } from '@prisma/client'
+import { BaseService } from '@/app/api/core/services/base.service'
+import { CopilotAPI } from '@/utils/CopilotAPI'
+import User from '../core/models/User.model'
```

Comment:
absolute import

Reply:
updated in 8a5aa1189856051dfb36da72b39c54ae45411ede commit